app/Http/Controllers/SupplyController.php

```diff
@@ -100,9 +100,9 @@ public function find(
     public function findScript(Request $request): StreamedResponse
     {
         $params = [
-            json_encode(config('app.adserver_host')),
-            json_encode(config('app.aduser_external_location')),
-            json_encode('.'.config('app.website_banner_class')),
+            config('app.adserver_banner_host'),
+            config('app.aduser_external_location'),
+            '.'.config('app.website_banner_class'),
```

Comment:
Hmmm.. I think that `adserver_host` and `adserver_banner_host` can be different addresses.
Adserver may store banners on EC2 or somewhere. In this situation, Adserver has a different address for his host and banners. This one (host) can be used for `register` path but adserver_banner_host can be used only during the creation of banner URL.

Reply:
done